src/templates/index.tsx

```diff
@@ -95,6 +96,6 @@ const Index = ({ data }: { data: any }) => {
 };
 
 export const render = (data: any) =>
-  reactWrapper(data, 'index', 'index.tsx', renderToString(<Index data={data} />), config.hydrate);
+  reactWrapper(data, 'index', 'index.tsx', renderToString(<Index data={data} />), config.hydrate, SchemaWrapper(data));
```

Comment:
I think the way we inject stuff like this into the head is going to change, but good to know that if we have to override stuff it's not too complicated.